examples/conftest.py

```diff
@@ -7,15 +7,13 @@
 import pandas as pd
 from packaging.version import Version
 
+# Setting this to not error out if no install is done.
+os.environ["PYTHONPATH"] = os.path.dirname(os.path.dirname(__file__))
+
 system = platform.system()
 py_version = sys.version_info[:2]
 PANDAS_GE_2_0_0 = Version(pd.__version__).release >= (2, 0, 0)
 
-# Having "OMP_NUM_THREADS"=1, set as an environment variable, can be needed
```

Comment:
This is currently set in the pixi_install task and no longer in the test.yaml itself.